### PR TITLE
Ajax scrolling implimented in place of pagination

### PR DESCRIPTION
**Pull request by tarunnjoshi**
Targets #2207

### Description
Ajax scrolling implemented in CRM->project

### Checklist:
- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Project/Resources/views/client-project.blade.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                        {{ $client->name }}
+                    </div>
+                    <div class="">{{ __('Total Hours Booked: ') . $client->current_hours_in_projects }}</div>
+                </div>
+            </td>
+        </tr>
+        @foreach($client->projects as $project)
+            <tr>
+                @can('projects.update')
+                    <td class="w-33p"><div class="pl-2 pl-xl-3"><a href="{{ route('project.show', $project) }}">{{ $project->name }}</a></div></td>
```
Comment (gautam071) [inline Modules/Project/Resources/views/client-project.blade.php:16]: formating

File: Modules/Project/Resources/views/client-project.blade.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                @else
+                    <td class="w-33p"><div class="pl-2 pl-xl-3">{{ $project->name }}</div></td>
```
Comment (gautam071) [inline Modules/Project/Resources/views/client-project.blade.php:18]: formating

File: Modules/Project/Resources/views/client-project.blade.php [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                    @endcan
+                <td class="w-20p">
+                    @foreach($project->getTeamMembers ?:[] as $teamMember)
+                    <span class="content tooltip-wrapper"  data-html="true" data-toggle="tooltip"  title="{{ $teamMember->user->name }} - {{ config('project.designation')[$teamMember->designation]}} <br>    Efforts: {{$teamMember->current_actual_effort}} Hours" >
+                        
+                        <a href={{ route('employees.show', $teamMember->user->employee) }}><img src="{{ $teamMember->user->avatar }}" class="w-35 h-30 rounded-circle mb-1 mr-0.5 {{ $teamMember->current_actual_effort >= $teamMember->current_expected_effort ? 'border border-success' : 'border border-danger' }} border-2"></a>
```
Comment (gautam071) [inline Modules/Project/Resources/views/client-project.blade.php:24]: Formating can you check use the blade formatter

**Review by gautam071 — CHANGES_REQUESTED** [review]

**Comment by tarunnjoshi** [comment]
Formatted

**Pull request closed without merging**